### ISSUE TITLE
jap file not imported with the new syntax

### ISSUE THREAD

**Issue by klag** [jolie/jolie#547]
I created a test for this bug here 

https://github.com/jolie/jolie/tree/bug/jap-file-new-syntax

**Comment by mwallnoefer** [comment]
@klag I have added two commits which should make it work.

**Comment by klag** [comment]
hi @mwallnoefer , there was a bug in the jap file, the line with from was commented replaced by an include.
I uploaded the fixed jap, and now the bug persists

**Comment by mwallnoefer** [comment]
Yeah, I had adjusted the JAP to match the pre-existing one containing the _twice_ example. I guess that `from` statements within a JAP are not working yet, you may only use `include`s.

**Comment by klag** [comment]
This is a bug. Jap are not useful if we cannot use from statement inside

Il Lun 23 Dic 2024, 19:50 Matthias Dieter Wallnöfer <
***@***.***> ha scritto:

**Comment by klag** [comment]
Did someone look into this? @kicito @fmontesi
